plugins/themes/default/styles/articleRevisions.less

```diff
@@ -0,0 +1,32 @@
+/**
+ * @file lib/pkp/styles/articleRevisions.less
```

Comment:
I created these 2 new javascript handlers:
js/pages/ArticleDownloadHandler.js
js/controllers/tab/issueEntry/SubmissionRevisionsHandler.js

Moreover, I made some additions to this file:
js/controllers/tab/issueEntry/IssueEntryTabHandler.js